### ISSUE TITLE
Category name limit is 40 chars... but Lunch Money allows longer names

### ISSUE THREAD

**Issue by casidiablo** [juftin/lunchable#127]
-   lunchable version:
-   Python version:
-   Operating System:

### Description

Got this error:

```
  File "/usr/local/lib/python3.12/site-packages/lunchable/models/categories.py", line 182, in get_categories
    CategoriesObject.model_validate(item) for item in categories
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/pydantic/main.py", line 551, in model_validate
    return cls.__pydantic_validator__.validate_python(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
pydantic_core._pydantic_core.ValidationError: 1 validation error for CategoriesObject
name
  String should have at most 40 characters [type=string_too_long, input_value='Sporting events amusement parks and museums', input_type=str]
    For further information visit https://errors.pydantic.dev/2.7/v/string_too_long
```

### What I Did

One of my categories has a long name in Lunch Money, which means they do support long cat names. But `lunchable` has a limit of 40 chars.

**Comment by justinflannery-sdx** [comment]
Ah, interesting. Their documentation lists a 40 character limit https://lunchmoney.dev/#categories

**Comment by juftin** [comment]
Ah, interesting. Their documentation lists a 40 character limit https://lunchmoney.dev/#categories

**Comment by casidiablo** [comment]
Yeah, I will ask in their discord what their actual limit is. Or maybe I can just brute force it in the UI

**Comment by casidiablo** [comment]
Tested in the UI. They support up to 100 chars.
<img width="337" alt="Screenshot 2024-11-11 at 11 32 10 AM" src="https://github.com/user-attachments/assets/6b2a7120-5621-4372-8123-b0cd37054a59">

**Comment by juftin** [comment]
This issue is resolved in `lunchable 1.4.1` - thanks for the patience!
